client/pages/favorites.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import Header from '../components/header';
+import ReactStars from 'react-rating-stars-component';
+
+export default class Favorites extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      favorites: [],
+      isModalOn: false,
+      isLoading: true
+    };
+    this.toggleOn = this.toggleOn.bind(this);
+    this.toggleOff = this.toggleOff.bind(this);
+  }
+
+  componentDidMount() {
+    fetch('/api/favorites', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json'
+      }
```

Comment:
You can leave this configuration out. The default method is GET and there is no body being sent in this request, so the Content-Type isn't needed.